### DISCUSSION TITLE
迷茫的业主看过来，这里有最新的消息。

### DISCUSSION THREAD

**Discussion by Hughlaud** [WeNeedHome/SummaryOfLoanSuspension#1018] [Ideas]
因为国内消息封锁比较严重，导致大家不知道这件事情到底产生了怎样的影响，上面的人究竟有没有着手去解决这件事情。
其实影响很大，李用了多种手段在解决，包括不限于：银行牵头的烂尾楼几百亿专项融资，在郑州试点的针对烂尾楼业主的廉租房等，以及对“停贷后究竟有没有风险”这一问题的分析和推测。
这里给大家推荐个youtube频道，播主是香港从业30年的财经类专业记者，每天都针对这件事情有专题节目，消息来源可靠：
https://www.youtube.com/watch?v=tyX7mfmmYeI
https://www.youtube.com/watch?v=trMKSER2tUo
https://www.youtube.com/watch?v=U_RnSuFxL6Q
频道名：王剑每日观察Kim's Observation

看过应该会有些头绪，希望能减轻大家一些焦虑。

**Comment by dxdleikai** [comment]
> 因为国内消息封锁比较严重，导致大家不知道这件事情到底产生了怎样的影响，上面的人究竟有没有着手去解决这件事情。 其实影响很大，李用了多种手段在解决，包括不限于：银行牵头的烂尾楼几百亿专项融资，在郑州试点的针对烂五楼业主的廉租房……等。 这里给大家推荐个youtube频道，播主是香港从业30年的财经类专业记者，每天都针对这件事情有专题节目，消息来源准备可靠： https://www.youtube.com/watch?v=tyX7mfmmYeI https://www.youtube.com/watch?v=trMKSER2tUo https://www.youtube.com/watch?v=U_RnSuFxL6Q
> 
> 看过应该会有些头绪，希望能减轻大家一些焦虑。

有没有能把视频down下来放到网盘里分享一下的

**Reply by Hughlaud** [reply]
找个梯子翻墙看吧，不难的。

**Reply by zxcs001** [reply]
找网盘分享你想多了，这东西传上去顶多两小时，就消失的不见了。
